### PR TITLE
Adding translation label and release drafter

### PR DESCRIPTION
**Pull request by leikoilja**
- Adding translation label
- Adding translation category to release drafter
- Adding translation labeler

### CODE REVIEW

File: .github/labeler.yml [hunk 1]
```diff
@@ -9,3 +9,7 @@ documentation:
 github structure:
   - any: [".github/**/*"]
     all: ["!**/*.py"]
+
+translation:
+  - any: ["custom_components/google_home/translations/*"]
+    all: ["!**/*.py"]
```
Comment (ArnyminerZ) [inline .github/labeler.yml:15]: Why adding this? Translations will always be in .json, don't they?

Comment (leikoilja) [reply .github/labeler.yml:15]: Yes they would, but I think it's better to set the explicit path to the `json` files so if we would to introduce any more `json` files and we change them, the GH won't mark them as "translation" label. For example, we have a file `hacs.json` but when that's modified we don't want to mark a PR with "translation" label :)

Comment (ArnyminerZ) [reply .github/labeler.yml:15]: I meant the `all` tag.

Comment (KapJI) [inline .github/labeler.yml:15]: I agree with @ArnyminerZ, this should be cleaner
```suggestion
  - any: ["custom_components/google_home/translations/*.json"]
```

**Review by ArnyminerZ — APPROVED** [review]

**Comment by leikoilja** [comment]
Awesome, thanks boys 🤩💥🔥

**Pull request merged**